### PR TITLE
add loginclass support for FreeBSD

### LINKED ISSUE #26381: user.present loginclass does not work on FreeBSD
The `user.present` documentation suggests that the `loginclass` argument is supported on BSD platforms. However, it does not appear to work on FreeBSD. A quick glance at the relevant source file (https://github.com/saltstack/salt/blob/develop/salt/states/user.py) reveals several instances of code checking specifically for `OpenBSD`. For example:

```python
    # OpenBSD login class
    if __grains__['kernel'] == 'OpenBSD':
        if not loginclass:
            loginclass = '""'
        if __salt__['user.get_loginclass'](name)['loginclass'] != loginclass:
            change['loginclass'] = loginclass
```

This code obviously isn’t going to work when `__grains__['kernel'] == 'FreeBSD'`, even though `loginclass` is an important feature on FreeBSD.

Perhaps the various `== 'OpenBSD'` checks could be replaced with a substring match for `BSD`?

For reference:

```yaml
# salt --versions-report
           Salt: 2015.5.3
         Python: 2.7.10 (default, Aug  9 2015, 01:20:33)
         Jinja2: 2.8
       M2Crypto: 0.22
 msgpack-python: 0.4.2
   msgpack-pure: Not Installed
       pycrypto: 2.6.1
        libnacl: Not Installed
         PyYAML: 3.11
          ioflo: Not Installed
          PyZMQ: 14.6.0
           RAET: Not Installed
            ZMQ: 4.1.2
           Mako: Not Installed
        Tornado: Not Installed
```

### PR DESCRIPTION
**Pull request by silenius**
I think it should also fix #26381

### CODE REVIEW

File: salt/modules/pw_user.py [hunk 1]
```diff
@@ -409,6 +436,23 @@ def info(name):
     return ret
 
 
+def get_loginclass(name):
+    '''
+    Get the login class of the user
```
Comment (jfindlay) [inline salt/modules/pw_user.py]: Will you add a `.. versionadded:: Boron` here?  Thanks.

**Comment by jfindlay** [comment]
@silenius, there is one [lint error](https://jenkins.saltstack.com/job/salt-pr-lint-n/8950/violations/file/salt/modules/pw_user.py/).

**Comment by silenius** [comment]
should be OK :)

**Comment by jfindlay** [comment]
Thanks, @silenius.

**Comment by cachedout** [comment]
Looks good. Thanks @silenius !

**Pull request merged**